opentelemetry-resource-detectors/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "opentelemetry-resource-detectors"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
The version would be incremented during release. Would be good to leave it as "0.1.0" for now.